test/cli_test.ts

```diff
@@ -561,4 +562,32 @@ describe('decaffeinate CLI', () => {
     `)
     );
   });
+
+  it('can wrap output in an IIFE', async () => {
+    await runCli(['--no-bare'], '', `(function() {\n\n}).call(this);`);
+  });
+
+  it('cannot use --modernize-js with --no-bare', async () => {
+    await runCli(
+      ['--no-bare', '--modernize-js'],
+      '',
+      '',
+      `
+      cannot use --modernize-js with --no-bare
```

Comment:
Just curious why `--modernize-js` cannot be used with `--no-bare`?

Reply:
I guess it's fine. I misremembered that `--modernize-js` implied `--use-js-modules`, but it doesn't. https://github.com/decaffeinate/decaffeinate/pull/2482